internal/lint/cr.go

```diff
@@ -76,6 +78,16 @@ func (s *ClusterRole) checkStale(ctx context.Context, refs *sync.Map) {
 		if s.system.skip(fqn) {
 			continue
 		}
+		partialRole := false
```

Comment:
nit: prefer using the zero value aka `var partialRole bool`

Reply:
sure, same functionality. I can change that